CMakeLists.txt

```
@@ -389,14 +414,21 @@ add_subdirectory(data)
 if (NOT MSVC)
     add_subdirectory(src/chkjson)
 endif()
-add_subdirectory(tests)
+
+if(TESTS)
+    add_subdirectory(tests)
+endif()
+
 if (JSON_FORMAT)
     add_subdirectory(tools/format)
 endif()
 if (CATA_CLANG_TIDY_PLUGIN OR CATA_CLANG_TIDY_EXECUTABLE)
     add_subdirectory(tools/clang-tidy-plugin)
 endif()
 
+install(DIRECTORY doc gfx
+    DESTINATION .
+)
```

This seems redundant with what's happening in data/CMakeLists.txt. Is it because data/CMakeLists.txt has the logic gated behind a `RELEASE` check? I don't understand why we need it when gfx is already handled? Should we deduplicate that logic to here?

Also, doc is developer docs and not needed for install either.

At the very least, should we do this?

> Use CMAKE_INSTALL_PREFIX as storage of data,gfx, etc.. Useful only on *nix OS.
```suggestion
install(DIRECTORY gfx
    DESTINATION ${CMAKE_INSTALL_PREFIX}
)
```

---

`DESTINATION .` is there becase _If a relative path is given it is interpreted relative to the value of the CMAKE_INSTALL_PREFIX variable._ [reference](https://cmake.org/cmake/help/v3.21/command/install.html?highlight=install#introduction)